claudetool/bash.go

```
@@ -247,6 +347,113 @@ func humanizeBytes(bytes int) string {
+		stdout.Close()
+		ptmx.Close()
+		return nil, fmt.Errorf("failed to write command to background pty: %w", err)
+	}
+
+	// Start a goroutine to copy pty output to the stdout file
+	go func() {
+		defer stdout.Close()
+		defer ptmx.Close()
+
+		// Copy all pty output to stdout file
+		io.Copy(stdout, ptmx)
```

golangci-lint (1.64.8)

Error return value of `io.Copy` is not checked

(errcheck)